recipes/mapseq/meta.yaml

```diff
@@ -1,16 +1,18 @@
-{% set version = "2.1.1" %}
-{% set sha256 = "b66af114f82ed9671e250995d014704b58b8e19b7ec95b1ef05a02eb8c5e8018" %}
+{% set version = "2.1.1a" %}
+{% set sha256 = "5a33c6ff169b34a9e6985f68e808172035a0f0cb5fea699f7ed997dd537d9fc1" %}
 
 package:
   name: mapseq
   version: '{{ version }}'
 
 source:
-  url: https://github.com/jfmrod/MAPseq/archive/v{{ version }}.tar.gz
+  url: https://github.com/meringlab/MAPseq/archive/{{ version }}.tar.gz
```

Comment:
Ok, I think this is ok. But please leave a travel in the old repo, in form of an issue for example, that the new forked version should be used.

Reply:
yea we will try to update the original repository.